linode/objkey/framework_model.go

```diff
@@ -65,14 +147,46 @@ func (rm *ResourceModel) CopyFrom(other ResourceModel, preserveKnown bool) {
 	rm.AccessKey = helper.KeepOrUpdateValue(rm.AccessKey, other.AccessKey, preserveKnown)
 	rm.SecretKey = helper.KeepOrUpdateValue(rm.SecretKey, other.SecretKey, preserveKnown)
 	rm.Limited = helper.KeepOrUpdateValue(rm.Limited, other.Limited, preserveKnown)
+	rm.Regions = helper.KeepOrUpdateValue(rm.Regions, other.Regions, preserveKnown)
+	rm.RegionDetails = helper.KeepOrUpdateValue(rm.RegionDetails, other.RegionDetails, preserveKnown)
+
 	if !preserveKnown {
 		rm.BucketAccess = other.BucketAccess
 	}
+
+	// rm.copyBucketAccessEntriesFrom(other.BucketAccess, preserveKnown)
 }
 
+// func (rm *ResourceModel) copyBucketAccessEntriesFrom(other []BucketAccessModelEntry, preserveKnown bool) {
+// 	if !preserveKnown {
+// 		rm.BucketAccess = other
+// 		return
+// 	}
+
+// if other == nil {
+// 	rm.BucketAccess = other
+// 	return
+// }
+
+// for i := range rm.BucketAccess {
+// 	if i > len(other) {
+// 		break
+// 	}
+// 	rm.BucketAccess[i].copyFrom(other[i], preserveKnown)
+// }
+// }
+
+// func (b *BucketAccessModelEntry) copyFrom(other BucketAccessModelEntry, preserveKnown bool) {
+// 	b.BucketName = helper.KeepOrUpdateValue(b.BucketName, other.BucketName, preserveKnown)
+// 	b.Cluster = helper.KeepOrUpdateValue(b.Cluster, other.Cluster, preserveKnown)
+// 	b.Permissions = helper.KeepOrUpdateValue(b.Permissions, other.Permissions, preserveKnown)
+// 	b.Region = helper.KeepOrUpdateValue(b.Region, other.Region, preserveKnown)
+// }
```

Comment:
Just double checking, is this safe to remove?

Reply:
This is related to https://discuss.hashicorp.com/t/behaviors-of-computed-attributes-within-a-block-during-update-ops/68339

Although they haven't responded, the Terraform behavior observed does support that these commented out code is not needed.